client/src/components/comparison/LeftArrow.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+const LeftArrow = (props) => {
+  return (
+    <div className='backArrow' onClick={this.props.goToPrevSlide}>
+      <i className='fa fa-angle-left fa-3x' aria-hidden='true'></i>
```

Comment:
These class names aren't super clear. They don't have to be perfectly clear, but they should give some indication of its use or elements that would be using it.

Reply:
put better names in the desc